plugins/nf-wave/src/main/io/seqera/wave/plugin/cli/WaveDebugCmd.groovy

```diff
@@ -47,40 +47,71 @@ class WaveDebugCmd {
     }
 
     protected void taskDebug(List<String> args) {
+        if( !args )
+            throw new AbortOperationException("Missing id of the task to debug or remote task path")
+
+        final criteria = args.pop()
+
+        if ( criteria.startsWith('s3://') ) {
```

Comment:
Considering we are going to introduce more backends, it would be better if it starts with a URI protocol. 

This should help https://github.com/nextflow-io/nextflow/blob/7d3cc707b575fa27c8844808257f85dbf56b2941/modules/nf-commons/src/main/nextflow/util/StringUtils.groovy#L35-L39